invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/search/components.js

```diff
@@ -109,6 +110,8 @@ export const RDMRecordSearchBarElement = withState(
     onInputChange,
     updateQueryState,
     currentQueryState,
+    showSharedDropdown = false,
+    mineLabel = "",
```

Comment:
To use the default of the child?
```suggestion
    mineLabel = undefined,
```